include/picongpu/fields/laserProfiles/PulseFromSpectrum.def

```diff
@@ -0,0 +1,135 @@
+/* Copyright 2013-2021 Axel Huebl, Heiko Burau, Anton Helm, Rene Widera,
+ *                     Richard Pausch, Alexander Debus
```

Comment:
please add your name too. If the files is not using comments from other files and you wrote all from scratch except the license header feel free to remove the other names.